ui/package.json

```diff
@@ -49,8 +49,6 @@
     "copy-webpack-plugin": "^7.0.0",
     "css-loader": "^5.1.1",
     "cypress": "^12.17.1",
-    "cypress-iframe": "^1.0.1",
```

Comment:
We're also not using cypress-iframe anymore, as the model register flow doesn't use the external SpaceTag nor whatever we did use for "The Templater".